pom.xml

```diff
@@ -1276,7 +1276,6 @@
             <skipJoramTests>false</skipJoramTests>
             <skipIntegrationTests>false</skipIntegrationTests>
             <skipIsolatedIntegrationTests>false</skipIsolatedIntegrationTests>
-            <skipCompatibilityTests>false</skipCompatibilityTests>
```

Comment:
You left it set not-skipped/enabled in the fast-tests profile.

Reply:
ouch! thanks!